### PR TITLE
Attempt to fix configlet lint issues

### PR DESCRIPTION
**Pull request by angelikatyborska**
Hi! This is my first contribution in this repo. The fact is that I don't know go at all 😅 but I want to try to help with the non-coding tasks and make the track ready for v3.

In this PR I was trying to get rid of all configlet lint issues, but that's not possible. There are a lot of concepts that weren't written yet, so they do not have authors. Empty authors is a linting error.

For concepts that had some text already, I tried to browse git history to figure the author, most likely the author of the exercise from which exercise's `after.md` was copied to concept's `about.md`.

### CODE REVIEW

File: concepts/conditionals/introduction.md [hunk 1]
```diff
@@ -1,46 +1,3 @@
-# Introduction
+# About
 
-Go contains basic numeric types that can represent sets of either integer or
```
Comment (angelikatyborska) [inline concepts/conditionals/introduction.md:3]: I'm not a go expert, but I think this document is about numbers, not conditionals, so I remvoed it.

**Comment by angelikatyborska** [comment]
@ekingery I'm taking a look at the track as promised. I cannot assign reviewers, so I'm pinging you instead

**Review by ekingery — APPROVED** [review]
Looks good, thank you!!

**Comment by angelikatyborska** [comment]
@ekingery Just a reminder that I have no write rights to this repository so I cannot merge the PR. I thought I mention this because I keep forgetting about it myself 😂. Unless we're waiting for more reviews or something else, then ignore my comment.

**Pull request merged**